include/ace/utils/custom.h

```diff
@@ -13,6 +13,11 @@
 // Here was __far attrib from DICE/GCC times. Not needed for VBCC, so it was removed.
 extern struct Custom custom;
 
+/**
+ * Bit value macro - useful for setting & testing bits.
+ */
+#define BV(x) (1 << (x))
```

Comment:
Should this be moved to more generic file?

Reply:
Done in 7ba50f5